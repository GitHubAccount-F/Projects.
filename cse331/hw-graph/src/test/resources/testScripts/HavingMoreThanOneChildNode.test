#Tests to see if a parent node can have multiple child nodes

#Creates empty graphs
CreateGraph graph1
CreateGraph graph2
CreateGraph graph3

#Adds multiple nodes
AddNode graph1 one
AddNode graph2 one
AddNode graph2 two
AddNode graph3 one
AddNode graph3 two
AddNode graph3 three

#Creates multiple edges
AddEdge graph2 one two e1
AddEdge graph3 one two e1
AddEdge graph3 one three e1

#Checks to see if there are multiple child nodes
ListChildren graph1 one
ListChildren graph2 one
ListChildren graph3 one