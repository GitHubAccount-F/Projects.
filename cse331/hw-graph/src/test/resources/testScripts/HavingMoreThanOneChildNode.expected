#Tests to see if a parent node can have multiple child nodes

#Creates empty graphs
created graph graph1
created graph graph2
created graph graph3

#Adds multiple nodes
added node one to graph1
added node one to graph2
added node two to graph2
added node one to graph3
added node two to graph3
added node three to graph3

#Creates multiple edges
added edge e1 from one to two in graph2
added edge e1 from one to two in graph3
added edge e1 from one to three in graph3

#Checks to see if there are multiple child nodes
the children of one in graph1 are:
the children of one in graph2 are: two(e1)
the children of one in graph3 are: three(e1) two(e1)